website/docs/guides/best-practices/environment-setup/2-one-deployment-environment.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "One deployment environment"
+id: 2-one-deployment-environment
+description: Learn how to configure environments in dbt Cloud.
+displayText: "dbt Cloud environment best practices"
+hoverSnippet: Learn how to configure environments in dbt Cloud.
+---
+
+
+## One deployment environment
+
+### **What this looks like**
+
+1. You have a **single *development* environment** where dbt users can access the dbt Cloud IDE and make changes to their code on feature branches created off of your default branch in your repository (most often the `main` branch)
+2. You have a **single *deployment* environment** (let’s call it “Production”) where your scheduled jobs run referencing the `main` branch
+3. You also have a [**Slim CI job**](docs/deploy/cloud-ci-job) that kicks off anytime you open a PR to merge a feature branch into `main`. This Slim CI job can run in your dbt “Production” environment
+
+:::info
+
+☁️ Slim CI jobs run in a dedicated custom schema for each PR, so there will no collision with your production schemas.
+
+:::
+
+![Table of basic setup for one deployment environment](/img/guides/best-practices/environment-setup/one-deployment-table.png)
+
+### `git` workflow
```

Comment:
```suggestion
### `git` workflow
```
this page has git in code blocks, the multi deployment page does not; I defer to not using code blocks in headers, so inclined to remove this and just call it "Git workflow"